ebu_tt_live/xsd1.1/ebutt_datatypes.xsd

```diff
@@ -27,7 +27,7 @@ Please note that the EBU-TT XML Schema is a helping document and NOT normative b
 	<xs:simpleType name="fontSizeType">
 		<xs:restriction base="xs:token">
 			<xs:pattern
-				value="(\d*\.?\d+(px|c|%))|([+-]?\d*\.?\d+(px|c|%))\s([+-]?\d*\.?\d+(px|c|%))"
+				value="([+-]?\d*\.?\d+(px|c|%))(\s([+-]?\d*\.?\d+(px|c|%)))?"
```

Comment:
I checked, it is defined the same in 3350. And indeed there is one line saying that it is `non-negative` lengthType. Maybe it would be good to modify the enhance the syntax with a `positiveLengthType` or something like that, because if like me you miss the line saying `non-negative lengthType` and you look directly at the syntax, you cannot guess that you are wrong